widget/popup.go

```diff
@@ -203,6 +203,7 @@ func (r *popUpRenderer) Refresh() {
 	if r.background.Size() != r.popUp.innerSize || r.background.Position() != r.popUp.innerPos {
 		r.Layout(r.popUp.Size())
 	}
+	// TODO should fix this refresh too?
```

Comment:
I don't know what this comment means, what is broken about the background refresh?

Reply:
I don't write it well, but it means if we should implemented in the same way as modalPopupRenderer.Refresh.

Reply:
@fpabl0 if you can think of a test which produces output that will change if you apply additional refresh here, then the answer to your question is “yes”.
Alternatively, it might be sufficient if the test which I requested with my review detects a change if you apply the refresh here.

Reply:
Ok, after some tests, popUpRenderer needs the same fix otherwise we are going to have the same issue as in dialog. So I will add popup tests too.

Reply:
done